config/runtimes/vllm-cpu-template.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     opendatahub.io/ootb: 'true'
   annotations:
     description: vLLM is a high-throughput and memory-efficient inference and serving engine for LLMs
-    openshift.io/display-name: vLLM CPU (ppc64le/s390x) ServingRuntime for KServe
+    openshift.io/display-name: vLLM CPU (ppc65le/s390x) ServingRuntime for KServe
```

Comment:
I believe it is `ppc64le` - needs to be corrected below too.
source: https://github.com/red-hat-data-services/vllm/blob/main/Dockerfile.ppc64le.ubi